test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Get_ApplicationJson_DictionaryObject_Tests.cs

```diff
@@ -84,5 +87,29 @@ public void Given_OpenApiDocument_Then_It_Should_Return_ComponentSchemaProperty(
 
             this._doc["components"]["schemas"][itemRef].Should().NotBeNullOrEmpty();
         }
+
+        [DataTestMethod]
+        [DataRow("string[]_string", "array")]
+        [DataRow("int32[]_int32", "array")]
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.